org.copyq.copyq.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id": "org.copyq.copyq",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "copyq",
+
+    "tag": "v3.2.0",
+    "commit": "ac9f67ee99caa072e85951ba9b6b6f89c1e0ee39",
+
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--env=XDG_CURRENT_DESKTOP=''"
+    ],
+
+    "modules": [
+        {
+            "name": "copyq",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hluk/CopyQ.git"
+                }
+            ],
+            "config-opts": [
+                "-DDESKTOP_INSTALL_NAME=org.copyq.copyq.desktop",
+                "-DAPPDATA_INSTALL_NAME=org.copyq.copyq.appdata.xml",
```

Comment:
You can't just rename the file like that. The `app-id` in the file is still `copyq.desktop` which is incorrect. Also this should not be build time configurable, your application should have one id that is always used everywhere.